modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -86,16 +86,7 @@ EOF
 $ oc get csv -n openshift-tempo-operator
 ----
 
-. Create a project of your choice for the TempoStack instance that you will create in a subsequent step:
-
-** To create a project from standard input without metadata:
-+
-[source,terminal]
-----
-$ oc new-project <project_of_tempostack_instance>
-----
-
-** To create a project from standard input with metadata:
+. Run the following command to create a project of your choice for the TempoStack instance that you will create in a subsequent step:
```

Comment:
@IshwarKanse, I had to move the table with supported storage secret parameters into a separate section, but I'd like to also remove one of two CLI options because we're using the other CLI option in the other steps. If you're okay with this, please just approve this PR.

Reply:
Thank you, Ishwar. I'm resolving this thread.